src/api/baggage.rs

```diff
@@ -193,3 +332,121 @@ impl BaggageExt for Context {
         self.get::<Baggage>().unwrap_or_else(|| &DEFAULT_BAGGAGE)
```

Comment:
Unrelated, but this could probably be:

```suggestion
        self.get::<Baggage>().unwrap_or(&DEFAULT_BAGGAGE)
```